svelte-kit-scss/src/lib/stores/navigation-progress.ts

```diff
@@ -0,0 +1,80 @@
+import { GITHUB_API_REQUEST_TIMEOUT_SECONDS } from '$lib/constants/github/github-api-request-timeout';
+import type { Navigation } from '@sveltejs/kit';
+import { get, readable, writable } from 'svelte/store';
+
+const LOCAL_STORAGE_KEY_ROUTE_LOAD_STATS = 'route_load_stats';
+
+export type RoutesLoadStats = { [routeId: string]: number[] };
+
+const getRouteLoadStats = (): RoutesLoadStats => {
+  const serialized = localStorage.getItem(LOCAL_STORAGE_KEY_ROUTE_LOAD_STATS);
+  const stats: RoutesLoadStats = serialized ? JSON.parse(serialized) : {};
+  return stats;
+};
+
+const GITHUB_API_REQUEST_TIMEOUT_MILISECONDS = GITHUB_API_REQUEST_TIMEOUT_SECONDS * 1000;
```

Comment:
Small typo, don't forget to update line 19 as well.

```suggestion
const GITHUB_API_REQUEST_TIMEOUT_MILLISECONDS = GITHUB_API_REQUEST_TIMEOUT_SECONDS * 1000;
```